### ISSUE TITLE
Amplify Auth additional node_modules folder after npm install

### ISSUE THREAD

**Issue by SRJames** [aws-amplify/amplify-js#3625]
I have an Angular application using Amplify to perform authentication using the Hosted UI.
It was all working until I performed an npm install.
After that, a call to Auth.currentSession() would error with "No userPool".
I found a similar issue [https://github.com/aws-amplify/amplify-js/issues/1323](url) that mentioned checking for an additional node_modules folder.
I found that there was a new node_modules folder under node_modules/@aws-amplify/auth.
Deleting this resolved the issue.
Could you provide any details on why this folder would have suddenly appeared?
Also, since the npm install this command
Auth.configure()  will not compile (using typescript in VSCode) as it now says that it requires 1 arguement. 
But the documentation [https://aws-amplify.github.io/docs/js/authentication#manual-setup](url) suggests this should work, so I am concerned that something else has gone wrong following the update.
Should this function still work as described in the documentation?

I have attached my package and package-lock files.
[package.txt](https://github.com/aws-amplify/amplify-js/files/3381911/package.txt)
[package-lock.txt](https://github.com/aws-amplify/amplify-js/files/3381912/package-lock.txt)

**Comment by gerrytsui** [comment]
This line in your package.json

>     "@aws-amplify/auth": "^1.2.25",

install this folder node_modules/@aws-amplify/auth

In our package.json, we do not install aws-amplify at all

here is what our package.json looks like

> ...
>     "@aws-amplify/analytics": "^1.2.18",
>     "@aws-amplify/api": "^1.0.37",
>     "@aws-amplify/auth": "^1.2.26",
>     "@aws-amplify/ui": "^1.0.20",
>     "aws-amplify-angular": "^3.0.5",
> ...

**Comment by SRJames** [comment]
Thanks for the reply. I've resolved this, it was a package issue. The @aws-amplify/auth package should not have been included, just aws-amplify and aws-amplify-angular as stated here [https://aws-amplify.github.io/docs/js/angular#installation](url)
I uninstalled all three, deleted the @aws-amplify folder from node_modules, reinstalled. 
I also changed the import for Amplify to
import Amplify from '@aws-amplify/core.

On the Auth.configure() problem. This doesn't work but Amplify.Auth.configure() does.
